sci-libs/minc/ChangeLog

```diff
@@ -0,0 +1,8 @@
+# ChangeLog for sci-libs/minc
+# Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
+# $Header: $
+
+*minc-2.2.00 (28 May 2015)
+
+  28 May 2015;  <chymera@gentoo.org> +metadata.xml, +minc-2.2.00.ebuild:
```

Comment:
Seriously, you should fix your local setup first before messing up more and more ChangeLog entries.